LayoutTests/imported/w3c/web-platform-tests/svg/animations/seeking-events-6-expected.txt

```diff
@@ -1,3 +1,3 @@
 
-FAIL Seeking backwards does not dispatch 'repeatEvent' assert_true: Not expecting event, but got beginEvent event expected true got false
+FAIL Seeking backwards does not dispatch 'repeatEvent' assert_equals: Expected beginEvent event, but got repeatEvent event instead expected "beginEvent" but got "repeatEvent"
```

Comment:
Does this mean we are dispatching events but we should not be? Is that something we can do right when adding `repeatEvent`, or does it need to be fixed separately?